fc/cli/watch.py

```diff
@@ -0,0 +1,27 @@
+import click
+from ..jira.issue import Issue
+from ..auth.combo import ComboAuth
+from . import cli_library
+
+
+@click.command()
+@click.option('--username')
+@click.argument('issue_ids', nargs=-1, metavar='ISSUE-1 ISSUE-2 ISSUE-3 etc.')
+def watch(username: str, issue_ids: tuple):
+    auth = ComboAuth(username)
+
+    for i in issue_ids:
```

Comment:
I'd like to use a better name than a single letter variable, `i`.  Perhaps the singular `issue_id`?

Reply:
Yeah I actually switched back to i to keep it concise, but I can agree to that.